lib/org/chromium/webidl/WebKitParser.js

```diff
@@ -33,6 +33,9 @@ foam.CLASS({
                       repeat0(alt(notChars('*'), seq('*', notChars('\/')))),
                       '*\/'),
                   // C-style preprocessor directives.
+                  seq(alt('#ifdef', '#ifndef'),
```

Comment:
This change was made while we were still processing the files in `win/interfaces`. Our current implementation only ignores one line preprocessor directives (such as `#define`). 

I have verified that this is not needed when only parsing files in `WebCore`